packages/api/cms-api/src/user-permissions/user-permissions.module.ts

```diff
@@ -25,15 +25,27 @@ export class UserPermissionsModule {
                     provide: USER_PERMISSIONS_OPTIONS,
                     useValue: options,
                 },
+                {
+                    provide: USER_PERMISSIONS_SERVICE,
+                    useClass: UserPermissionsService,
+                },
             ],
+            exports: [USER_PERMISSIONS_SERVICE],
         };
     }
 
     static forRootAsync(asyncOptions: UserPermissionsAsyncOptions): DynamicModule {
         return {
             module: UserPermissionsModule,
             imports: asyncOptions.imports,
-            providers: [this.createProvider(asyncOptions)],
+            providers: [
+                this.createProvider(asyncOptions),
+                {
+                    provide: USER_PERMISSIONS_SERVICE,
+                    useClass: UserPermissionsService,
```

Comment:
You are exporting UserPermissionsService twice I think

1. here, under the key USER_PERMISSIONS_SERVICE
2. in line 16 under the key UserPermissionsService

I think there should only be 1.

Reply:
763cfa12af87d57ad52bf16d26bc5fd3b1f531b2

Reply:
You are still exporting twice: Line 17 vs. line 23

(you "only" removed the (minimal) code duplication)

Reply:
why export it as `USER_PERMISSIONS_SERVICE` at all, using just `UserPermissionsService` (as before) should be fine? (as it's not injectable - or is that planned?)